src/components/Chart.js

```diff
@@ -24,9 +23,9 @@ const Chart = ({ stock, latestPrice, updateChartRange }) => {
 
     return (
         <div className="chart">
-            {stock.length === 0
-                ? <div className="loading-spinner"></div>
-                : <>
+        {(stock === null) ? <div className="loading-spinner"></div> :
```

Comment:
I would also highly recommend making a new React component called `<LoadingSpinner />` which renders this `<div>` with the classname and associated styling, etc.
Otherwise it's time consuming if you ever need to change it (e.g. when you use emotion)

Reply:
good idea, will try this way